package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "react-side-effect",
-  "version": "2.1.2",
+  "version": "3.0.0",
   "description": "Create components whose prop changes map to a global side effect",
   "main": "lib/index.js",
   "scripts": {
     "build": "node scripts/build.js",
     "clean": "rimraf lib",
     "prepare": "npm test && npm run clean && npm run build",
-    "test": "mocha",
-    "test:watch": "mocha --watch",
-    "test:cov": "babel-node ./node_modules/.bin/isparta cover ./node_modules/.bin/_mocha"
+    "test": "jest --forceExit",
```

Comment:
there are open handles now and i didn't have enough time to figure out why. It's the node MessageChannel class